client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -319,6 +324,47 @@ const getVisiblePlugins = ( plugins, country, industry, productTypes ) => {
 	);
 };
 
+const transformRemoteExtensions = ( extensionData, localeValue ) => {
+	return extensionData.reduce( ( result, extension ) => {
+		const localeData =
+			extension.locales.find(
+				( locale ) => locale.locale === localeValue
+			) ||
+			extension.locales.find( ( locale ) => locale.locale === 'en_US' );
+		const transformedExtension = {
+			title: localeData.title,
+			slug: extension.key,
+			description: generatePluginDescriptionWithLink(
+				localeData.description,
+				extension.key
+			),
+			isVisible: () => true,
+		};
+		const sectionIndex = extension.section === 'primary' ? 0 : 1;
+		result[ sectionIndex ].plugins.push( transformedExtension );
+		return result;
+	}, initialInstallableExtensions );
```

Comment:
This should potentially be changed to `[ ...initialInstallableExtensions ]` as it will otherwise modify `initialInstallableExtensions` and continue to push items to that list every time `transformRemoteExtensions` is called.
Just switching between the `Business details` and `Free features` tab causes this:
<img width="565" alt="Screen Shot 2021-05-10 at 12 17 00 PM" src="https://user-images.githubusercontent.com/2240960/117682979-fdb3ff80-b189-11eb-8a9f-2cb14e4429e5.png">

Reply:
Thanks @louwie17, thats good spotting! Fixed in https://github.com/woocommerce/woocommerce-admin/pull/6952/commits/6ab3594cde9e10aff8d2a9fc95ec6dff5d6f4bec